### PR TITLE
Added flag to cockroach demo for disabling rangefeeds

### PR DESCRIPTION
**Pull request by kathancox**
[Fixes DOC-4702](https://cockroachlabs.atlassian.net/browse/DOC-4702)

Added the auto enable rangefeed flag to the `cockroach demo` page.

### CODE REVIEW

**Review by HonoreDB — APPROVED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14636)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @kathancox)

File: v22.1/cockroach-demo.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ The `demo` command supports the following general-use flags.
 
 Flag | Description
 -----|------------
+`--auto-enable-rangefeeds` | Override the default behavior of `cockroach demo`, which has rangefeeds enabled on startup. Use `--auto-enable-rangefeeds=false` to disable rangefeeds — this will improve performance. To run a [changefeed](create-and-configure-changefeeds.html) on a demo cluster, you must enable rangefeeds. See [Enable rangefeeds](use-changefeeds.html#enable-rangefeeds) for more detail. <br><br>**Default:** `true`
```
Comment (andf-crl) [inline v22.1/cockroach-demo.md:113]: nit: Consider mentioning "rangefeeds required for changefeeds" a bit earlier, maybe something like:

```suggestion
`--auto-enable-rangefeeds` | Override the default behavior of `cockroach demo`, which has rangefeeds enabled on startup.  If you do not need to use [changefeeds](create-and-configure-changefeeds.html) with your demo cluster, use `--auto-enable-rangefeeds=false` to disable rangefeeds and improve performance. See [Enable rangefeeds](use-changefeeds.html#enable-rangefeeds) for more detail. <br><br>**Default:** `true`
```

**Review by andf-crl — APPROVED** [review]
LGTM with one optional clarity recommendation. Thanks!

**Comment by kathancox** [comment]
TFTRs Andrew & Aaron!

**Pull request merged**